src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -234,6 +236,39 @@ private static bool IsFirewallEnabled()
                 return false;
             }
         }
+
+        private async Task<string> GetPsVersionInfo(IExecutionContext executionContext)
+        {
+            var builder = new StringBuilder();
+
+            string powerShellExe = HostContext.GetService<IPowerShellExeUtil>().GetPath();
+            string arguments = @"Write-Host ($PSVersionTable | Out-String)";
+            using (var processInvoker = HostContext.CreateService<IProcessInvoker>())
+            {
+                processInvoker.OutputDataReceived += (object sender, ProcessDataReceivedEventArgs args) =>
+                {
+                    executionContext.Output($"STDOUT: {args.Data}");
```

Comment:
just builder.AppendLine(args.Data); for both STDOUT/STDERR.
we don't need this to executionContext.Output().

Reply:
Done. Ready for review.